index.html

```diff
@@ -7,27 +7,45 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Figma Project</title>
   <link rel="stylesheet" href="./style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
 
 <body>
-  <div class="first_page">
-    <header class="header flex_container">
-      <div class="logo">ReMotely</div>
-      <nav class="nav flex_container">
-        <div class="flex_container"><a href="#">book</a><a href="#">explore</a><a href="#">about</a></div>
-        <div><a href="#">sign in</a></div>
-      </nav>
-    </header>
 
 
-    <div class="catch_phrase">
-      <h1>Work from a Cabin near Helsinki</h1>
-      <button class="btn" type="submit">Explore</button>
-      <img class="main_photo" src="switzphoto.jpg" alt="chatka ze Szwajcarii">
+  <header class="header flex_container">
+    <div class="logo">ReMotely</div>
+    <nav class="nav flex_container">
+      <div class="link_flex_container"><a class="links" href="#">book</a><a class="links" href="#">explore</a>
+        <a class="links" href="#">about</a>
+      </div>
+      <div class="sign_container"><a class="sign" href="#">sign in</a></div>
+    </nav>
+  </header>
+
+  <section class="first_page">
+    <div class="column phrase">
+      <div class="catch_phrase">
+        <h1>Work from <br> a Cabin near Helsinki</h1>
+      </div>
     </div>
+    <div class="column cottage">
+      <div class="photo">
+        <img class="main_photo" src="switzphoto.jpg" alt="chatka ze Szwajcarii">
+      </div>
+  </section>
   </div>
+  </div>
+  <div class=button_container>
+    <button class="btn" type="submit"><span class="exp">Explore</span></button>
+  </div>
+
```

Comment:
```suggestion
  <section class="first_page">
    <div class="column phrase">
      <div class="catch_phrase">
        <h1>Work from <br> a Cabin near Helsinki</h1>
      </div>
    </div>
    <div class="column cottage">
      <div class="photo">
        <img class="main_photo" src="switzphoto.jpg" alt="chatka ze Szwajcarii">
      </div>
    </div>
      <div class=button_container>
    <button class="btn" type="submit"><span class="exp">Explore</span></button>
  </div>  
  </section>
```

Reply:
Just copy it and paste inside VSCode

Reply:
if i put this button into the section it will ruin whole my code I did it and somehow button switched its position on top...